File: Web/kashish12_kashishalikhan_2125cse1133_02/GettingStarted/gcdandlcm.cpp
#include <iostream>
using namespace std;
int main(){
    int num1, num2;
    cin >> num1 >> num2;
    int t1,t2,gcd,lcm;
    t1=num1;
    t2=num2;
    while (num1 % num2 != 0)
    {
      int remainder = num1 % num2;
      num1 = num2;
      num2 = remainder;
    }
    gcd = num2;
    lcm = (t1 * t2) / gcd;
    cout<<gcd<<endl<<lcm;
    return 0;
}

File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/GcdAndLcm.java
import java.util.*;
public class GcdAndLcm {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        sc.close(); 
        int l,r,a,b;
        a=num1;
        b=num2;
        while(a%b>0)
        {
            r=a%b;
            a=b;
            b=r;
        }
        System.out.println(b); //gcd
        l= (num1*num2)/b;
        System.out.println(l); //lcm
       }
}
